src/plots/cartesian/axes.js

```diff
@@ -774,9 +788,6 @@ axes.calcTicks = function calcTicks(ax, opts) {
             _minor // noSuffixPrefix
         );
 
-        if(isPeriod && prevText === t.text) continue;
```

Comment:
Maybe better to merge the repetitive labels together later in the process, instead of skipping.